apps/files/src/actions/editLocallyAction.ts

```diff
@@ -43,7 +98,7 @@ export const action = new FileAction({
 	},
 
 	async exec(node: Node) {
-		openLocalClient(node.path)
+		startOpenLocalProcess(node.path)
```

Comment:
```suggestion
		attemptOpenLocalClient(node.path)
```
The `startOpenLocalProcess` wrapper seems unnecessary, would call attempt directly